wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -262,7 +262,8 @@ def get_searchable_content(self, value):
 
 class RichTextBlock(FieldBlock):
 
-    def __init__(self, required=True, help_text=None, **kwargs):
+    def __init__(self, required=True, help_text=None, widget=None, **kwargs):
+        self.widget = widget
```

Comment:
I think would be better to set all field options to `self.field_options` (as already done with `help_text` and `required`).

If you decided to do so, `field` method can look similar to this:
```python
    @cached_property
    def field(self):
        field_options = self.field_options.copy()

        if 'widget' not in field_options:
            from wagtail.wagtailcore.fields import RichTextArea
            field_options.update({
                'widget': RichTextArea
            })

        return forms.CharField(**field_options)
```

Another good solution is set all field options to properties in `RichTextBlock`, but this is not widely used in Wagtail: as I see only [`ChooserBlock`](https://github.com/torchbox/wagtail/blob/v1.3.1/wagtail/wagtailcore/blocks/field_block.py#L334) done this way. If you decided to go this way, you can deal with widget like this done in Django. Refs:
* https://github.com/django/django/blob/1.9.1/django/forms/fields.py#L59
* https://github.com/django/django/blob/1.9.1/django/forms/fields.py#L103